bot.user.js

```diff
@@ -461,8 +461,7 @@ var bot = (function() {
                         var hCircle = {
                             x: window.snakes[snake].xx,
                             y: window.snakes[snake].yy,
-                            radius: window.getSnakeWidth(window
-                                .snakes[snake].sc)
+                            radius: window.getSnakeWidth(window.snakes[snake].sc) / 2
```

Comment:
Seeing as we use this idiom a lot, maybe it would be better to create a getSnakeRadius function... I'll create an issue for it.

Thanks for making the function more accurate to its name!